### PR TITLE
add custom emoji slider

### PR DESCRIPTION
**Pull request by shivamlivelike**

### CODE REVIEW

File: app/src/main/res/values/colors.xml [hunk 1]
```diff
@@ -7,5 +7,10 @@
     <color name="quiz_incorrect_result_bar_color">#FF3C3C</color>
     <color name="quiz_correct_result_bar_color">#00FF78</color>
     <color name="quiz_default_result_bar_color">#E6E6E6</color>
+    <color name="image_slider_widget_result_center_color">#D10D0D</color>
+    <color name="image_slider_widget_result_end_color">#0650C3</color>
+    <color name="image_slider_gradient_start">#140099</color>
```
Comment (mittalshivansh) [inline app/src/main/res/values/colors.xml:12]: there should be no gradient as per designs shared, am I missing something?

Comment (shivamlivelike) [reply app/src/main/res/values/colors.xml:12]: there is a gradient of track of slider as per seen in zaplin

Comment (mittalshivansh) [reply app/src/main/res/values/colors.xml:12]: ok, how you have handled the result gradient? can you pinpoint me to there?

Comment (shivamlivelike) [reply app/src/main/res/values/colors.xml:12]: i am not sure of that ,it is the same code that you used for creating default imageslider maybe you can check for that

**Pull request merged**